appveyor.yml

```diff
@@ -0,0 +1,24 @@
+version: 0.1.{build}
+pull_requests:
+  do_not_increment_build_number: true
+
+environment:
+  matrix:
+    - PYTHON: C:\Python27-x64
+    - PYTHON: C:\Python36-x64
+
+install:
+  - ps: >
+      & "$env:PYTHON\python.exe" -m virtualenv C:\pacifica;
+      C:\pacifica\Scripts\activate.ps1;
+      python -m pip install --upgrade pip setuptools wheel;
+      python -m pip install -r requirements-dev.txt
+
+build: off
+
+test_script:
+  - ps: >
+      mkdir C:\tmp; C:\pacifica\Scripts\activate.ps1;
+      pre-commit run -a;
+      coverage run --include='pacifica/*' -m pytest -v;
+      coverage report -m --fail-under=100;
```

Comment:
I wasn't aware we used Appveyor. My understanding is that it is for projects running in Windows - is there a requirement that this application run in a Windows environment?

Reply:
What's the origin of that requirement? Depending on what we're doing, supporting Windows servers with the same code base as *nix servers can be costly.

Reply:
We can discuss costs of supporting Windows as a platform in a different issue. Perhaps in the core pacifica/pacifica repository and discussion out of that issue can lead to a pacifica-docs repository issue to solidify support requirements.

Reply:
Sorry, I still don't understand what the origin of the requirement is. Is this coming from management? Is it coming from you? You've created this requirement in a project that I'm responsible for developing before I've even written any code, and I'd like to understand what the rationale of that decision is because I'm not sure I agree with it.

Reply:
The requirement comes from the community, in order to reach the most possible users to show that the code can be supported on different platforms.

I'm wondering if you are getting hung up on what's expected of you to develop the code. I don't expect you to develop and test the code on a Windows platform before you commit. I expect that  someone else in the community will need to help you to support all the platforms. If you can show coverage and testing on one platform (Travis) the other can be brought along later.